src/layouts/core/Core.component.js

```diff
@@ -1,10 +1,11 @@
 import * as R from 'ramda';
 import React from 'react';
-import { Header, Footer, data, utils } from '@pingcap/pingcap-ui';
+import { Header, Footer, getData, utils } from '@pingcap/pingcap-ui';
 import { useRouter } from 'next/router';
 
 import { link as linkUtils } from 'utils';
 
+const data = getData(/^https?:\/\/(?:tug\.)?tidb\.io/);
```

Comment:
It is possible to base on the bashPath provided by `useHistory`? Because the Core layout may be also used for a different domain other than `tug.tidb.io` and we'd better make it as flexible as we could.

https://nextjs.org/docs/api-reference/next/router#router-object